libs/R-TypeLogic/Global/Components/LayerLvL.hpp

```diff
@@ -17,7 +17,7 @@ namespace ecs
     class LayerLvL : public Component {
       public:
         /// @brief Enumeration of the different layers can be used
-        enum layer_e { UNKNOWN, BACKGROUND, MIDDLE, DECORATION, OBSTACLE, BUTTON, WRITABLE, ENEMY, PLAYER, PROJECTILE, LAYER_NUMBER };
+        enum layer_e { UNKNOWN, BACKGROUND, MIDDLE, DECORATION, OBSTACLE, BUTTON, WRITABLE, WRITABLE_BUTTON, ENEMY, PLAYER, PROJECTILE, LAYER_NUMBER };
```

Comment:
You should not, if think writable is enough

Reply:
Refer to discord conversation